### PR TITLE
[#8943] Update user map as at Jun 2018

### LINKED ISSUE #8943: Update user map as at Jun 2018
To be done after #8814 

Covers new users from May-June 2018

New entries to consider:

Country| University
-- | --
Australia | Deakin University
Australia | Dungog High School
Australia | Monash University
Australia | RMIT University
Australia | University of Technology Sydney
Bangladesh | American International University-Bangladesh
Belgium | Universiteit Hasselt
Brazil | Universidade Anhembi Morumbi
Brazil | Universidade Federal da Bahia
Brazil | Universidade Federal de São João del-Rei
Canada | Argyle School
Canada | Laval University
Canada | Schulich School of Business, York University
Canada | Simon Fraser University
Chile | Universidad Adolfo Ibañez
Chile | USM
China | TsingHua Universtiy High School International
Cyprus | University of Cyprus
Hong Kong | HKU School of Professional and Continuing Education
Hong Kong | The University of Hong Kong
Hong Kong | The University of Hong Kong
India | JK Lakshmipat University
Indonesia | Universitas Pendidikan Indonesia
Korea | Soongduk Middle School
Malaysia | ASIA SCHOOL OF BUSINESS
Malaysia | Universiti Teknologi Malaysia
Malaysia | Universiti Tenaga Nasional
Malaysia | University Malaya
Mexico | CETYS
Mexico | ITESM
Netherlands | Avans University of applied sciences
Netherlands | Fontys
Netherlands | Hogeschool Utrecht
Netherlands | HvA
Netherlands | mboRijnland
Netherlands | Saxion University of Applied Sciences
New Zealand | University of Auckland
New Zealand | University of Otago
Oman | Sultan Qaboos University
Portugal | University of Minho
Portugal | University of Trás-os-Montes e Alto Douro
Russia | Tomsk State University
Singapore | Institute of Technical Education
Singapore | Weave
Spain | IE University
Thailand | King Mongkut's University of Technology Thonburi
UK | East Tennessee State University
UK | Kingston University
UK | Plymouth University
UK | Royal Veterinary College
UK | Swansea University
UK | University of Bath
UK | University of Bradford
UK | University of Portsmouth
UK | University of Sheffield
USA | American International School of Utah
USA | Atlantic Technical College & Technical High School
USA | Belmont University
USA | Brigham Young University - Idaho
USA | California Polytechnic State University
USA | California State University, Fresno
USA | California State University, Long Beach
USA | Central Carolina Community College
USA | Central Connecticut State University
USA | Central Michigan University
USA | College America
USA | College of the Holy Cross
USA | Cornell University
USA | Drexel University
USA | duPont Manual High School
USA | East Tennessee State University
USA | El-Abbadi Charter Academy
USA | Frontier Nursing University
USA | Gann Academy
USA | Georgia Institute of Technology
USA | Gulliver Schools
USA | Immaculata Catholic School
USA | Ithaca College
USA | Ithaca College
USA | Kansas State University
USA | Kennesaw State University
USA | Lawrence Technological University
USA | Lincoln-Sudbury Regional High School
USA | Louisiana State University
USA | Maryville University St. Louis
USA | Methuen High School
USA | Milwaukee School of Engineering
USA | Mounds View Public Schools
USA | Nazareth College
USA | New York Institute of Technology
USA | Northeastern University
USA | Ohio University
USA | Purdue University
USA | Saline High School
USA | Santa Clara University
USA | St. Philip's College, Alamo Colleges District
USA | Thales University
USA | The University of North Carolina at Charlotte
USA | UC Berkeley
USA | University at Buffalo
USA | University of California Irvine
USA | University of California, Los Angeles
USA | University of California, San Diego
USA | University of Florida
USA | University of Hartford
USA | University of Illinois, Urbana-Champaign
USA | University of Iowa
USA | University of Michigan
USA | University of Minnesota
USA | University of North Carolina at Chapel Hill
USA | University of South Alabama
USA | University of Southern California
USA | University of Texas Rio Grande Valley
USA | University of the Incarnate Word
USA | Virginia Commonwealth University
USA | West Virginia University
USA | LECOM
USA | Miami Dade College - Kendall
USA | Plymouth University
USA | Uniformed Services University

### PR DESCRIPTION
**Pull request by ccannawright**
Fixes #8943 

**Outline of Solution**

It seems past attempts to finish this were closed from inactivity, unless I'm misunderstanding - this is my first contribution

### CODE REVIEW

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -186,6 +194,7 @@
       "Algonquin College",
       "Appleby College",
       "Aussie Pet Mobile Groomer Training",
+      "Argyle School",
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: `Argyle Secondary School` instead

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -123,6 +127,7 @@
       "Talentenschool Turnhout",
       "Thomas More Vorselaar",
       "Universiteit Gent",
+      "Universiteit Hasselt",
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: can change to `University of Hasselt`? according to google

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -459,14 +475,13 @@
...
       "Elsa High School",
-      "HKU",
       "HKU SPACE Community College",
       "HKUST",
       "Queen Elizabeth School",
       "The Chinese University of Hong Kong",
       "The Hong Kong Polytechnic University",
       "The ISF Academy",
-      "University of Hong Kong",
+      "University of Hong Kong (HKU)",
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: may want to add `The` in front

File: src/main/webapp/data/userMapData.json [hunk 4]
```diff
@@ -498,6 +513,7 @@
       "IIT Madras",
       "IMT Ghaziabad",
       "Indian Institute of Technology",
+      "JK Lakshmipat University",
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: may want to put below `Jadavpur University` instead?

File: src/main/webapp/data/userMapData.json [hunk 5]
```diff
@@ -843,6 +868,7 @@
       "Instituto Superior Tecnico",
       "Musaic",
       "School of Education Porto",
+      "Universidade de Trás-os-Montes e Alto Douro",
       "Universidade do Minho"
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: Can help to change this to `University of Minho` instead

File: src/main/webapp/data/userMapData.json [hunk 6]
```diff
@@ -1188,6 +1221,7 @@
       "Ashland University",
       "Ashton Elementary",
       "Astor School",
+      "Atlantic Technical Collage & Technical High School",
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: typo. I think `Atlantic Technical College` will do

File: src/main/webapp/data/userMapData.json [hunk 7]
```diff
@@ -1575,6 +1623,7 @@
       "Marshall B. Ketchum University",
       "Mary Ellen Henderson",
       "Marywood University",
+      "Maryville University St. Louis",
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: can delete this. it is the same as `Maryville University`

File: src/main/webapp/data/userMapData.json [hunk 8]
```diff
@@ -1539,8 +1584,10 @@
       "Lane Community College",
       "Language Learning Center",
       "Las Positas College",
+      "Lawrence Technological University",
       "Lebanon High School",
       "Lebanon Valley College",
+      "LECOM",
```
Comment (jacoblipech) [inline src/main/webapp/data/userMapData.json]: better to put `The Lake Erie College of Osteopathic Medicine (LECOM)`

File: src/main/webapp/data/userMapData.json (hunk 2 above)
Comment (ccannawright) [reply src/main/webapp/data/userMapData.json]: I was confused on this point which relates to one below: should we be putting the names as they are spelled in the language of the country they're from? Or should we translate them all to the English version of the name? I looked at a bit closer and it looks like on their dutch site they only use the name UHasselt, wikipedia says it was rebranded in 2005. The english site says Hasselt University.

Comment (damithc) [reply src/main/webapp/data/userMapData.json]: For consistency, we can use the English equivalent. I think some of the existing entries might not follow that policy, but fixing that can be a separate PR.

File: src/main/webapp/data/userMapData.json (hunk 7 above)
Comment (jacoblipech) [reply src/main/webapp/data/userMapData.json]: I think you missed out on this! Both names direct to the same university so I think can remove.

Comment (ccannawright) [reply src/main/webapp/data/userMapData.json]: Where is the other Maryville? There's Marywood but that's not the same University

Comment (jacoblipech) [reply src/main/webapp/data/userMapData.json]: ohh i see. My bad on this!

**Comment by jacoblipech** [comment]
@ccannawright Can you resolve the merge conflict? Thanks!

**Comment by wkurniawan07** [comment]
Looks like this is affected by the infrastructure upgrade done two days ago.
I will help you to resolve it. `userMapData.json` is now located at `src/web/data`.

**Comment by RonakLakhotia** [comment]
I didn't see this PR was open, however #9343 resolves the conflicts caused due to the migration.

**Pull request merged**